src/components/input/input-variables.scss

```diff
@@ -0,0 +1,27 @@
+$input-container-padding: 2px !default;
```

Comment:
This filename should start with an underscore to indicate that it's a partial

Reply:
Agreed. I did that at first, but then noticed other files not doing it, so I reverted for consistency. I'll see about updating them all.